packages/@aws-cdk/aws-appmesh/lib/tls-listener.ts

```diff
@@ -33,4 +34,12 @@ export interface TlsListener {
    * The TLS mode.
```

Comment:
I understand your concern and you got a point. I will change it to `ListenerTlsConfig` for now.

I am worried to use`Options` for suffix since we already use that to represent different group of objects. I am mimicking the [`ConnectionPoolConfig`](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-appmesh/lib/private/utils.ts#L12).

Reply:
I would still go for `ListenerTlsOptions`. It's a convention we use in many places in the CDK, I'm not worried it will be too confusing.

I wouldn't call it `ListenerTlsConfig` - that actually means something different in the CDK (the result of executing `bind()`).

Reply:
Well noted. I am convinced to make this change by adding the `Options` suffix.

**One last follow up question:**
To be consistent across the table, do you think other properties like `HealthCheck` or OutlierDetection` should be also renamed by adding the `Options` suffix? (*I won't be making those change in this PR)

Reply:
Nah. It's fine.